client/signup/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -35,81 +37,87 @@ export default React.createClass( {
 		}
 		if ( this.props.noticeType === 'isDotCom' ) {
 			noticeValues.icon = 'block';
-			noticeValues.text = this.translate( 'That\'s a WordPress.com site, so you don\'t need to connect it.' );
+			noticeValues.text = translate( 'That\'s a WordPress.com site, so you don\'t need to connect it.' );
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'notWordPress' ) {
 			noticeValues.icon = 'block';
-			noticeValues.text = this.translate( 'That\'s not a WordPress site.' );
+			noticeValues.text = translate( 'That\'s not a WordPress site.' );
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'notActiveJetpack' ) {
 			noticeValues.icon = 'block';
-			noticeValues.text = this.translate( 'Jetpack is deactivated.' );
+			noticeValues.text = translate( 'Jetpack is deactivated.' );
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'outdatedJetpack' ) {
 			noticeValues.icon = 'block';
-			noticeValues.text = this.translate( 'You must update Jetpack before connecting.' );
+			noticeValues.text = translate( 'You must update Jetpack before connecting.' );
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'jetpackIsDisconnected' ) {
 			noticeValues.icon = 'link-break';
-			noticeValues.text = this.translate( 'Jetpack is currently disconnected.' );
+			noticeValues.text = translate( 'Jetpack is currently disconnected.' );
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'jetpackIsValid' ) {
 			noticeValues.status = 'is-success';
 			noticeValues.icon = 'plugins';
-			noticeValues.text = this.translate( 'Jetpack is connected.' );
+			noticeValues.text = translate( 'Jetpack is connected.' );
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'notJetpack' ) {
 			noticeValues.status = 'is-noticeType';
 			noticeValues.icon = 'status';
-			noticeValues.text = this.translate( 'Jetpack couldn\'t be found.' );
+			noticeValues.text = translate( 'Jetpack couldn\'t be found.' );
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'alreadyConnected' ) {
 			noticeValues.status = 'is-success';
 			noticeValues.icon = 'status';
-			noticeValues.text = this.translate( 'This site is already connected!' );
+			noticeValues.text = translate( 'This site is already connected!' );
+			noticeValues.showDismiss = false;
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/13599/commits/a49e5f4d7c945d8057587dac709a2d67fa260d04.